app/src/main/res/values/strings.xml

```diff
@@ -60,123 +60,123 @@
+    Not protected.
     </string>
 
     <string name="dns_explanation_dns_connected">Connected to RethinkDNS</string>
-    <string name="dns_explanation_firewall_connected">Connected to Firewall</string>
+    <string name="dns_explanation_firewall_connected">Connected to firewall</string>
 
     <string name="dns_explanation_connected" description="The message will be shown when the DNS is connected">
-        Connected to RethinkDNS and Firewall
+        Connected to RethinkDNS and firewall
     </string>
 
     <string name="universal_firewall_explanation">
-        Apply firewall rules on all applications based on device events, for example, firewall when the device is locked or when a particular app is not in use.
+        Apply firewall rules on all apps based on device events, for example, firewall when the device is locked or when a particular app is not in use.
     </string>
 
-    <string name="notification_action_resume_vpn">RESUME</string>
-    <string name="notification_action_pause_vpn">PAUSE</string>
-    <string name="notification_action_stop_vpn">STOP</string>
+    <string name="notification_action_resume_vpn">Resume</string>
+    <string name="notification_action_pause_vpn">Pause</string>
+    <string name="notification_action_stop_vpn">Stop</string>
     <string name="notification_action_dns_mode">DNS</string>
-    <string name="notification_action_dns_firewall_mode">DNS + Firewall</string>
+    <string name="notification_action_dns_firewall_mode">DNS + firewall</string>
     <string name="dns_mode_info_title">DNS</string>
 
-    <string name="firewall_rules">Network Rules</string>
+    <string name="firewall_rules">Network rules</string>
 
     <string name="firewall_mode_info_title">Firewall</string>
     <string name="firewall_mode_info_explanation_new">Monitor network activity and firewall any app or IP address.</string>
 
     <string name="apps_info_title">Apps</string>
 
-    <string name="hybrid_mode_info_title">DNS + Firewall</string>
+    <string name="hybrid_mode_info_title">DNS + firewall</string>
 
     <string name="dns_mode_info_explanation_new">Monitor, block, and encrypt DNS requests.</string>
 
     <string name="dns_firewall_mode_info_explanation_new">All of the above + bypass Internet censorship.</string>
     <string name="wireguard_mode_info_explanation_new">Coming soon.</string>
 
-    <string name="alert_permission_accessibility">Permission - Accessibility</string>
+    <string name="alert_permission_accessibility">Permission — Accessibility</string>
     <string name="alert_permission_accessibility_regrant">Action Required</string>
-    <string name="alert_firewall_accessibility_explanation">RethinkDNS uses accessibility service to track which apps are in-use (foreground) versus those that are not in-use (background).\n\nRethinkDNS does not collect any information or spy on you in any way.</string>
```

Comment:
Many "in-use" vs. "in use". Not sure what that does, but made them all "in use".